config-thunderjet.yml

```diff
@@ -0,0 +1,70 @@
+okapi-url: http://localhost:9130
+registry-url: http://folio-registry.dev.folio.org
+install-json-path: /home/x/projects/folio/busybee/platform-complete/install.json  # A file path to install.json can also be used
+env-vars: # any key-pairs added here will be added to the OKAPI env service
+  DB_PASSWORD: folio_admin
+  DB_USERNAME: folio_admin
+  DB_DATABASE: okapi_modules
+  DB_HOST: host.docker.internal
+  DB_PORT: 5432
+  OKAPI_URL: http://host.docker.internal:9130
+  KAFKA_HOST: host.docker.internal
+  KAFKA_PORT: 9092
+  ELASTICSEARCH_URL: http://host.docker.internal:9200
+  SYSTEM_USER_PASSWORD: system_user_password
+  AWS_URL: http://host.docker.internal:9000
+  AWS_REGION: us-east-1
+  AWS_BUCKET: example-bucket
+  AWS_ACCESS_KEY_ID: minioadmin
+  AWS_SECRET_ACCESS_KEY: minioadmin
+  S3_IS_AWS: false
+  S3_URL: http://host.docker.internal:9000                                                                                                                                                                       
+  S3_REGION: us-east-1                                                                                                                                                                                                                                                                                                                                   
+  S3_ACCESS_KEY_ID: minioadmin                                                                                                                                                                  
+  S3_SECRET_ACCESS_KEY: minioadmin
+  S3_BUCKET: example-bucket
+  SPLIT_FILES_ENABLED: true
+be-modules: # list the FOLIO backend modules you would like to spin up
+  ## CORE: ensure that core FOLIO modules are first on this list
+  - mod-permissions
+  - mod-configuration
+  - mod-login
+  - mod-users
+  - mod-users-bl
+  - mod-authtoken
+  ## END CORE
+  - mod-source-record-manager
+  - mod-source-record-storage
+  - mod-inventory
+  - mod-inventory-storage
+  - mod-organizations
+  - mod-organizations-storage
+  - mod-entities-links
+  - mod-invoice
+  - mod-invoice-storage
+  - mod-orders
+  - mod-orders-storage
+  - mod-pubsub
+  - mod-tags
+  - mod-finance
+  - mod-finance-storage
+  - mod-circulation-storage
+  - mod-circulation
+  ## Additional
+  - mod-audit
+  - mod-template-engine
+  - mod-feesfines
+  ## Consortia
+  - mod-consortia # add this module to install.json file
+  - mod-password-validator
+
+ui-modules:
+  - folio_users
+  - folio_data-import
+  - folio_tenant-settings
+  - folio_organizations
+  - folio_inventory
+  - folio_invoice
+  - folio_orders
+  - folio_finance
+  - folio_receiving
```

Comment:
if we add folio_consortia_settings - we will be able to run consortia UI also.
But we can do it in subsequent PR

Reply:
For the UI the following are also useful:
- in be-modules: mod-agreements
- in ui-modules: folio_acquisition-units, folio_developer

Reply:
most guys will only need consorita and order/finance/invoice. So I will include as comment